common/unit_test/Test_Common_Sorting.hpp

```diff
@@ -53,7 +53,7 @@
 #include <KokkosKernels_Utils.hpp>
 #include <KokkosKernels_Sorting.hpp>
 #include <KokkosKernels_default_types.hpp>
-#include <KokkosSparse_CrsMatrix.hpp>
+// #include <KokkosSparse_CrsMatrix.hpp>
```

Comment:
This line can just be gone right? SortCrs is now its own header in sparse and it's also unit tested in the sparse component.

Reply:
Yeah, I'll clean that up